ui/src/project.ts

```diff
@@ -0,0 +1,54 @@
+import { writable } from "svelte/store";
+
+import * as api from "./api";
+import * as event from "./event";
+import * as remote from "./remote";
```

Comment:
I prefer named imports; I feel they communicate everything upfront and put the compiler errors in a single place if something is missing from the module:
`import { createStore } from "./remote"`

Reply:
I'm in favour of qualification as it usually ends up that the module name is repeated in the names of methods and types, i.e.: `createStore` -> `createRemoteStore`. And if not it helps to contextualize and give indication that it is a foreign type.